src/Support_Endpoints/Plugins.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Plugins listing endpoint for Support Hub
+ */
+
+namespace Avidly\Support_Client\Support_Endpoints;
+
+class Plugins extends Support_Endpoint {
+	public function __construct() {
+		$this->route = '/plugins/';
+		$this->register();
```

Comment:
The register call should be elsewhere, or called outside the class. The purpose of the constructor is to initialize the class and not to execute code.